elastic/kibana
# [Inventory] Entity names redirect on click to respective pages

## Summary

Adds the ability to click through to the overview pages for entities on the Entity Name cell for the Entity Grid on the new Inventory page.

https://github.com/user-attachments/assets/e712d3ef-370f-4353-a398-2365176eb582

Closes #192676 

### How to test

- Go to Inventory Page.
- Click on an Entity Name.

**Expected**: Should redirect to the overview page of that Entity, regardless it is a `host`, `container`, or `service`.
